envoy/upstream/upstream.h

```diff
@@ -1163,8 +1163,9 @@ namespace fmt {
 template <> struct formatter<Envoy::Upstream::Host> : formatter<absl::string_view> {
   template <typename FormatContext>
   auto format(const Envoy::Upstream::Host& host, FormatContext& ctx) -> decltype(ctx.out()) {
-    absl::string_view out =
-        host.hostname().empty() ? host.hostname() : host.address()->asStringView();
+    absl::string_view out = host.hostname().empty() ? host.hostname()
+                            : host.address()        ? host.address()->asStringView()
```

Comment:
Not your code but shouldn't this be: ?

```suggestion
    absl::string_view out = !host.hostname().empty() ? host.hostname()
                            : host.address()        ? host.address()->asStringView()
```